frontend/src/pages/Notifications/NotificationsSnackbar.js

```diff
@@ -66,9 +71,9 @@ const SnackbarContentWrapper = withStyles(styles)(ContentWrapper);
 
 const NotificationsSnackbar = props => {
   return (
-    <Snackbar open={props.showSnackbar} autoHideDuration={4000} onClose={props.closeSnackbar}>
+    <Snackbar open={props.showSnackbar} autoHideDuration={4000000} onClose={props.closeSnackbar}>
```

Comment:
4000000 ms is roughly one hour, is this intentional?

Reply:
Wasn't intentional, I thought I changed it back